sdk/storage/azure-storage-file/azure/storage/file/_generated/operations/_file_operations.py

```diff
@@ -382,9 +440,19 @@ def delete(self, timeout=None, cls=None, **kwargs):
             return cls(response, None, response_headers)
     delete.metadata = {'url': '/{shareName}/{directory}/{fileName}'}
 
-    def set_http_headers(self, timeout=None, file_content_length=None, file_http_headers=None, cls=None, **kwargs):
+    def set_http_headers(self, file_attributes="none", file_creation_time="now", file_last_write_time="now", timeout=None, file_content_length=None, file_permission="inherit", file_permission_key=None, file_http_headers=None, cls=None, **kwargs):
```

Comment:
It seems the default value for those new added parameters should be 'Preserve'?

Reply:
Your swagger shares the same parameter definition for Create and SetProperties.  The default value for Create is `Now` and the default value for SetProperties is `Preserve`.  We can patch up the swagger at some point, but for now we're passing `Preserve` in the methods that call SetProperties.